website/docs/sdk-reference/rust.mdx

```diff
@@ -0,0 +1,689 @@
+
+**Number-based comparators** (`=`, `<`, `>=`, etc.)
+* accept `Int`, `UInt`, or `Float` values,
+* accept `String` values containing a properly formatted, valid `Float` value,
+* all other values are considered invalid (a warning will be logged and the currently evaluated targeting rule will be skipped).
+
+**Date time-based comparators** (`BEFORE` / `AFTER`)
+* accept `DateTime` values, which are automatically converted to a second-based Unix timestamp,
+* accept `Int`, `UInt`, or `Float` values representing a second-based Unix timestamp,
+* accept `String` values containing a properly formatted, valid `Float` value,
+* all other values are considered invalid (a warning will be logged and the currently evaluated targeting rule will be skipped).
+
+**String array-based comparators** (`ARRAY CONTAINS ANY OF` / `ARRAY NOT CONTAINS ANY OF`)
+* accept `Vec` of `String`s,
+* accept `String` values containing a valid JSON string which can be deserialized to an array of `String`,
+* all other values are considered invalid (a warning will be logged and the currently evaluated targeting rule will be skipped).
+
+### Default user
+
+It's possible to set a default User Object that will be used on feature flag and setting evaluation. It can be useful when your application has a single user only or rarely switches users.
+
+You can set the default User Object on SDK initialization:
+
+```rust
+let client = Client::builder("#YOUR-SDK-KEY#")
+    // highlight-next-line
+    .default_user(User::new("john@example.com"))
+    .build()
+    .unwrap();
+```
+
+Whenever the evaluation methods like `get_value()`, `get_value_details()`, etc. are called without an explicit `user` parameter, the SDK will automatically use the default user as a User Object.
+
+```rust
+let client = Client::builder("#YOUR-SDK-KEY#")
+    .default_user(User::new("john@example.com"))
+    .build()
+    .unwrap();
+
+// The default user will be used in the evaluation process.
+let value = client.get_value("keyOfMyFeatureFlag", None, false).await;
+```
+
+When a `user` parameter is passed to the evaluation methods, it takes precedence over the default user.
+
+```rust
+let client = Client::builder("#YOUR-SDK-KEY#")
+    .default_user(User::new("john@example.com"))
+    .build()
+    .unwrap();
+
+let other_user = User::new("brian@example.com");
+
+// other_user will be used in the evaluation process.
+let value = client.get_value("keyOfMyFeatureFlag", Some(other_user), false).await;
+```
+
+## Polling Modes
+
+The _ConfigCat SDK_ supports 3 different polling mechanisms to acquire the setting values from _ConfigCat_. After latest setting values are downloaded, they are stored in the local cache, then all `get_value()` calls are served from there. With the following polling modes, you can customize the SDK to best fit to your application's lifecycle.
+[More about polling modes.](../advanced/caching.mdx)
+
+### Auto polling (default)
```

Comment:
The polling mode parameters (autoPollIntervalInSeconds, cacheRefreshIntervalInSeconds) are described in the other SDKs documentation, i think it would be nice to have this here as well.

Reply:
Added a few sentences about the `Duration` parameters